ebu_tt_live/xsd/ebutt_d.xsd

```diff
@@ -21,7 +21,7 @@
 		
 		<xs:sequence>
 			<xs:element ref="ttm:copyright" minOccurs="0"/>
-			<xs:element name="metadata" type="tt:d_metadata_type" minOccurs="0">
+			<xs:element name="metadata" type="ebuttm:headMetadata_type" minOccurs="0">
```

Comment:
I'm happy to go ahead with this for now, noting that we should open an issue to refactor the XSDs a bit anyway to bring the LIT up to spec with the latest EBU-TT Part 1 and EBU-TT-D versions.